File: 00.for循环/main.py
import datetime

start = datetime.datetime.now()
print(start)

sum = 0
len = 20000 #
for i in range(len):
  for j in range(len):
    sum += 1

end = datetime.datetime.now()
print(end)

print(sum)

print((end - start).microseconds)

File: 00.for循环/Main.java
public class Main {
  public static void main(String[] args) {
    long start = System.currentTimeMillis();

    long sum = 0;
    // long length = 20000; // 136ms
    // for (long i = 0; i < length; i++) {
    //   for (long j = 0; j < length; j++) {
    //     sum++;
    //   }
    // }

    for (long i = 0; i < 1000000000l; i++) {
      sum++;
    }

    long end = System.currentTimeMillis();
    System.out.println(sum);
    System.out.println((end - start) + "ms");
  }
}
